lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,27 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # Describes car's state and behaviour
+  class Car
+    attr_accessor :year, :color, :model, :current_speed
+
+    def self.default_car
+      new
+    end
+
+    def initialize(args)
+      @year = args.fetch(:year, 2000)
```

Comment:
@vbyno, yes I have. But tests says that initialize should take an hash of options